packages/ra-ui-materialui/src/list/pagination/PaginationActions.tsx

```diff
@@ -13,6 +13,8 @@ export const PaginationActions: FC<PaginationActionsProps> = memo(props => {
         onPageChange,
         size = 'small',
         className,
+        // @ts-ignore
+        slotProps,
```

Comment:
No, add it to the rest props sanitizer instead (cf https://github.com/marmelab/react-admin/pull/9474/files)